web/src/components/UserPart/UserPart.js

```diff
@@ -0,0 +1,18 @@
+function UserPart({ userName, partName }) {
+  return (
+    <h3 className="text-xl">
+    <span className=" border-2 border-purple-900 mr-1"></span>
+      <span className="text-gray-500">
+        {userName}
+      </span>
+      <span className="text-2xl mx-1 text-gray-300">
```

Comment:
I would like this slash to be aligned at the top and bottom with the indigo bar at the start of the line,
and so I think you should probably use the same technique there and use skew.
https://tailwindcss.com/docs/skew#app.
It should also be gray-400